README.md

```diff
@@ -269,3 +273,30 @@ defined in.
  */
 #define functionMacro( x, y ) ( x + y )
 ```
+## Branching
+
+### Procedure
+1. In your local repo run ```git checkout alpha```
+2. run ```git checkout -b "<your branch name>"```
+3. The first time you push the branch there will an error. Just follow the instructions to set the upstream branch.
+
+### Naming
+All branches **MUST** follow the branch naming scheme. Any branch that does not follow the structure will not be allowed to be pushed.
+The structure is as follows:
+- ft-YYYY-branch_name
+
+Where YYYY is one of the 4 letter module codes:
+
+- MNGT = Management. This is just a general tag for any team management task like restructuring the code base or formatting doxygen code docs
+- BOOT = Bootloader module
+- MAIN = Main Application module
+- COMM = Comms processing module
+- IMAG = Image processing module
+- PAYL = Payload processing module
+- HOUS = Housekeeping module
+
+The branch name is **all** lowercase with words separated by underscores.
+
```

Comment:
Be more specific here that the branch_name portion is lowercase, and that YYYY is uppercase. It's stated later as well, but the more explicit the better. Even if you put "make sure this is all uppercase" on line 288 that could work fine too. Otherwise looks great!